cli/lib/platforms/cordova/util.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const fs = require('fs');
+const cfg = require('../../cfg');
+const log = require('../../log');
+const path = require('path');
+const Promise = require('bluebird');
+const spawnPromise = require('spawn-rx').spawnPromise;
+
+/**
+ * Executes a Cordova command, e.g "platform add ios"
+ * @param   {Array<String>} args
+ * @return  {Promise<String>}
+ */
+const exec = exports.exec = function exec (args) {
+  log.debug('executing cordova command with args - %s', args.join(', '));
+  // Promise.resolve will create a Bluebird promise chain
+  return Promise.resolve()
+    .then(() => spawnPromise(resolveCordovaLocation(), args));
+};
+
+
+/**
+ * We might need to resolve strings such as "local" or "global" to an actual path
+ * @return {String}
+ */
+function resolveCordovaLocation () {
```

Comment:
This uses the setting chosen by the user on `init` to determine which cordova binary to use.